frontend/src/components/HeaderComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+import { Navbar, NavbarBrand, Nav, NavItem, Button } from "reactstrap";
+
+const HeaderComponent = (props) => {
+  const handleLogOut = (event) => {
+    let confirmlogout = window.confirm("Are you sure you wish to logout?");
+    if (confirmlogout) {
+      alert("User Logged Out");
+      fetch("http://localhost:3000/users/logout")
```

Comment:
Try to use constant here and then it is easy to change those ports and hosts.

Reply:
ok ayye